storage/migrations/16_consensus_tx_index.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+CREATE INDEX ix_transactions_method ON chain.transactions (method);
```

Comment:
this kind of index usually ends up needing `height` at the end, otherwise the api can't get "recent transactions of x method" without walking this index for all transactions of a method and sorting them or ignoring this index and scanning chronologically and filtering by method

Reply:
Good point, will update 👍

Reply:
Do you think It might make sense to update the `ix_transactions_sender` index in the same way?

Most of the accounts probably have a relatively low number of transactions which are fine, but there are likely some accounts with huge amount of transactions which could cause problems for the "recent transactions of x sender" query, since it would need to sort the filtered list of transactions which could be large?

Likely way less problematic, I should probably find the accounts with most transactions and test the queries on a live nexus deployment.

Reply:
Adding the `ix_transactions_sender_block` index fixes the above examples to be quick.